contrib/pyln-proto/pyln/proto/message/bolts/Makefile

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/make
+
+SPECDIR := ../../../../../../../lightning-rfc
```

Comment:
Should probably be expressed in function of `git rev-parse --show-toplevel` to make it stable and a bit clearer which directory is intended.